utils/machine/info.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
```

Comment:
Because it already existed :)  I moved the whole `utils/machine` package to a top-level `machine` package (we should reconsider where this package should live when https://github.com/google/cadvisor/issues/1250 is implemented)